docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -43,7 +43,7 @@ Antenatal care attendance module
 ++++++++++++
 
 This module determines whether or not a simulant attends an antenatal care visit according to their ANC visit 
```

Comment:
```suggestion
This module determines a simulant's antenatal care attendance exposure according to their ANC visit 
```
Proposing this change to go along with @alibow's [proposed changes for the next line](https://github.com/ihmeuw/vivarium_research/pull/1695/files#r2169501236), since the module determines not just *whether* the simulant attends ANC, but *when*, and we're collectively calling this "ANC attendance exposure," I think.

I think the original wording was unclear because it looked like it was saying we use the probability (or propensity) *and* timing to determine whether a simulant attends ANC, but I think the intention was to say that we use the propensity to determine ANC attendance and timing.